install/windows/build.py

```diff
@@ -101,34 +94,35 @@ def build_delphiviewer_inst():
     elif i == 2:
         build_odemisviewer_inst()
         add_size_to_version()
-    elif i == 3:
-        build_delphiviewer_exe()
-    elif i == 4:
-        build_delphiviewer_inst()
-        add_size_to_version()
-    elif i == 5:
-        build_odemisviewer_exe()
-        build_delphiviewer_exe()
-    elif i == 6:
-        build_odemisviewer_inst()
-        build_delphiviewer_inst()
-        add_size_to_version()
-    elif i == 7:
-        build_odemisviewer_exe()
-        build_odemisviewer_inst()
-        add_size_to_version()
-    elif i == 8:
-        build_delphiviewer_exe()
-        build_delphiviewer_inst()
-        add_size_to_version()
+#     elif i == 3:
+#         build_delphiviewer_exe()
+#     elif i == 4:
+#         build_delphiviewer_inst()
+#         add_size_to_version()
+#     elif i == 5:
+#         build_odemisviewer_exe()
+#         build_delphiviewer_exe()
+#     elif i == 6:
+#         build_odemisviewer_inst()
+#         build_delphiviewer_inst()
+#         add_size_to_version()
+#     elif i == 7:
+#         build_odemisviewer_exe()
+#         build_odemisviewer_inst()
+#         add_size_to_version()
+#     elif i == 8:
+#         build_delphiviewer_exe()
+#         build_delphiviewer_inst()
+#         add_size_to_version()
     elif i == 9:
```

Comment:
Either undelete this function or add a comment why it should remain commented and not deleted

Reply:
Indeed, keeping all the code was a little bit over the top. I've kept the most useful code, so that it's easy to build again the Delphi Viewer if we ever need it, and added a comment about this.